components/encrypt-signup.jsx

```diff
@@ -42,7 +46,7 @@ module.exports = React.createClass({
     }
     request.onreadystatechange = () => {
       if (request.readyState === 4) {
-        // TODO: do something when submitted
+        ga.event({category: "Signup", action: "Submitted the form"});
```

Comment:
We can add this, but I realized that these events were being fired even if the POST event failed. I think our callback/error checking needs work here